tests/manage/pv_services/test_overprovision_level_policy_control_with_capacity.py

```diff
@@ -0,0 +1,208 @@
+import logging
+import pytest
+
+from ocs_ci.ocs import defaults
+from ocs_ci.ocs.resources.storage_cluster import verify_storage_cluster
+from ocs_ci.ocs.ocp import OCP
+from ocs_ci.ocs import constants
+from ocs_ci.framework.testlib import (
+    ManageTest,
+    tier1,
+    bugzilla,
+    skipif_managed_service,
+)
+
+log = logging.getLogger(__name__)
+
+
+@pytest.fixture(autouse=True, scope="class")
+def setup_sc(storageclass_factory_class):
+    sc_fs_obj = storageclass_factory_class(
+        interface=constants.CEPHFILESYSTEM, sc_name="sc-test-fs"
+    )
+    sc_blk_obj = storageclass_factory_class(
+        interface=constants.CEPHBLOCKPOOL, sc_name="sc-test-blk"
+    )
+    return {
+        constants.CEPHBLOCKPOOL_SC: None,
+        constants.CEPHFILESYSTEM_SC: None,
+        "sc-test-blk": sc_blk_obj,
+        "sc-test-fs": sc_fs_obj,
```

Comment:
I'm not sure if cephfs is supported. Need to check with dev

Reply:
Removed Filesystem specific code for now. Once we confirm from dev then i will add it again.